### PR TITLE
Ore Visual Change

### PR DESCRIPTION
**Pull request by Alekshhh**
## About the PR
Changed ores visually. Plasma and silver from vg. Quartz from goon and reshaded by me. The rest are mine. Adds a faint glow to plasma, uranium and bananium

## Media

image
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:Alekshhh
- tweak: Changed ores visually.

### CODE REVIEW

**Comment by UbaserB** [comment]
hooray

File: Resources/Prototypes/Entities/Objects/Materials/ore.yml [hunk 1]
```diff
@@ -95,6 +95,10 @@
   - type: PhysicalComposition
     materialComposition:
       RawPlasma: 500
+  - type: PointLight
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Materials/ore.yml:98]: add `castShadows: false` to all of these ores

**Review by EmoGarbage404 — APPROVED** [review]
gud sprites

**Comment by TheShuEd** [comment]
lmao, braifood just did an animation on the old ore sprites #23633

**Pull request merged**